dash_bio/component_factory/_volcano.py

```diff
@@ -355,13 +389,18 @@ def figure(
             col = 'black'
 
         layout = go.Layout(
-            title=title,
+            title={'text': 'Volcano Plot',
```

Comment:
Should we be using the `title` arg here? As it is we're ignoring it, right? Though passing the `title` arg directly to `title.text` in the layout means you can't use the dict form of `title` to modify its styling. Then the only way to do that would be to use magic underscores in the kwargs (`title_font_color="red"` etc).

Alternatively we could drop `title` as an arg completely, so it gets picked up by `layout.update`. The downside of that approach is that passing in a string for `title` would wipe out the default style you have below - at least until https://github.com/plotly/plotly.py/issues/3148 gets fixed, which hopefully will not be too long (🙏  @nicolaskruchten )

So yeah, I guess my inclination is to drop `title` as an arg, even though it's a breaking change in the short term, because the current behavior will be restored with much more flexibility after we fix https://github.com/plotly/plotly.py/issues/3148

Reply:
Dropped in 3e2ce66.